build/Glyssen.proj

```diff
@@ -211,12 +211,18 @@
 	depending on whether your project's output MUST be signed or it's only nice if it is. For example, a continuous build
 	might use SignIfPossible, while a Publication build must use SignInstaller.-->
 	<Target Name="SignInstaller" DependsOnTargets="VersionNumbers; Installer">
-		<Exec Command='sign "$(RootDir)\output\installer\$(InstallerFileName)"'></Exec>
-		<Message Text="Signed installer at $(RootDir)\output\installer\$(InstallerFileName)" Importance="high"/>
+		<Exec Command='insignia -ib "$(InstallerPathWithVersion)" -o engine.exe'></Exec>
+		<Exec Command='sign engine.exe'></Exec>
+		<Exec Command='insignia -ab engine.exe "$(InstallerPathWithVersion)" -o $(InstallerPathWithVersion)'></Exec>
+		<Exec Command='sign "$(InstallerPathWithVersion)"'></Exec>
```

Comment:
Did you see https://wixtoolset.org/documentation/manual/v3/overview/insignia.html? That suggests that you could do it in the .wixproj file by overriding two targets there.

Reply:
I looked at it. Seems promising, but I'm a little sketchy on the details. Specifically, I don't know what our "sign" command does. My guess is that it's set up on the build agent to call **SignTool sign** with the appropriate command-line parameters. Also, I think that would then require some different plumbing to get ContinueOnError set correctly. I did take a stab at it, but I can't even get it to attempt to sign. I tried setting the **SignOutput** to true in both wixproj files (for the MSI and for the bootstrapper), but it doesn't seem to change anything so maybe I'm doing something wrong.

Reply:
See https://github.com/sillsdev/Glyssen/pull/671